frontend/packages/dapple/config.addresses.json

```diff
@@ -48,7 +48,8 @@
       "SNGLS": "0xaec2e87e0a235266d9c5adc9deb4b2e29b54d009",
       "VSL": "0x5c543e7ae0a1104f78406c340e9c64fd9fce5170",
       "PLU": "0xd8912c10681d8b21fd3742244f44658dba12264e",
-      "MLN": "0xbeb9ef514a379b997e0798fdcc901ee474b6d9a1"
+      "MLN": "0xbeb9ef514a379b997e0798fdcc901ee474b6d9a1",
+      "RHOC" : "0x168296bb09e24a88805cb9c33356536b980d3fc5"          
```

Comment:
I tried to look for RHOC somehow on kovan but I couldn't find one. I will check it with the guys from Rchain